
# HG changeset patch
# User Stuart Rackham <srackham@methods.co.nz>
# Date 1216675102 -43200
# Node ID eb3a27e2a48254a67790178fb70e9def8181bf49
# Parent 03ceadcba36c70b814701969a9548e82c95fedf4
Suppression of manpage references list now works again.

--- a/docbook-xsl/manpage.xsl	Mon Jul 14 16:14:47 2008 +1200
+++ b/docbook-xsl/manpage.xsl	Tue Jul 22 09:18:22 2008 +1200
@@ -21,6 +21,9 @@
 </xsl:template>
 
 <!-- Don't automatically generate the REFERENCES section -->
+<xsl:template name="endnotes.list">
+</xsl:template>
+<!-- Next entry for backward compatibility with old docbook-xsl versions -->
 <xsl:template name="format.links.list">
 </xsl:template>
 

